sensor_msgs/Image image
---
string caption
